src/diffusers/loaders/peft.py

```diff
@@ -216,7 +216,9 @@ def load_lora_adapter(self, pretrained_model_name_or_path_or_dict, prefix="trans
 
             rank = {}
             for key, val in state_dict.items():
-                if "lora_B" in key:
+                # Cannot figure out rank from lora layers that don't have atleast 2 dimensions.
+                # Bias layers in LoRA only have a single dimension
+                if "lora_B" in key and val.ndim > 1:
```

Comment:
Normally, we don't train the lora_B bias, but the control loras have trained biases. In this case, the assumption of being able to index `val.shape[1]` is incorrect since bias is a ndim=1 tensor